fendermint/testing/contract-test/src/ipc/subnet.rs

```diff
@@ -50,4 +56,104 @@ impl<DB: Blockstore> SubnetCaller<DB> {
             c.join(public_key.into()).from(addr).value(deposit)
         })
     }
+
+    /// Try to join the subnet as a validator.
+    pub fn try_join(
+        &self,
+        state: &mut FvmExecState<DB>,
+        validator: &Validator<Collateral>,
+    ) -> TryCallResult<()> {
+        let public_key = validator.public_key.0.serialize();
+        let addr = EthAddress::new_secp256k1(&public_key)?;
+        let deposit = from_fvm::to_eth_tokens(&validator.power.0)?;
```

Comment:
I like this `from_fvm` and `from_eth` conversion utils. I feel we should expose them as a crate and make them available to the `ipc` repo. I feel we have a lot of redundant conversion functions here and there :) 

//cc @cryptoAtwill, thoughts?

Reply:
They are exposed in the `messages` crate, but some low level ones might be useful on their own for someone who doesn't want to deal with Fendermint message types.